analyzers/src/SonarAnalyzer.CSharp/Rules/PropertyToAutoProperty.cs

```diff
@@ -173,8 +163,7 @@ private static bool TryGetFieldFromGetter(AccessorDeclarationSyntax getter, Sema
         {
             getterField = null;
 
-            var returnedExpression = GetReturnExpressionFromBody(getter.Body)
-                ?? GetReturnExpressionFromExpressionBody(getter.ExpressionBody());
+            var returnedExpression = GetReturnExpressionFromBody(getter.Body) ?? GetReturnExpressionFromExpressionBody(getter.ExpressionBody());
```

Comment:
After addressing other comments, the flow of this method can be simplified a lot. Also by renaming it properly and changing the result type `IFieldSymbol FieldSymbolFromGetter`